sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -135,8 +134,9 @@ class SQLQueryTestSuite extends QueryTest with SharedSparkSession {
     //   1. Maven can't get correct resource directory when resources in other jars.
     //   2. We test subclasses in the hive-thriftserver module.
     val sparkHome = {
-      assert(sys.props.contains("spark.test.home") ||
-        sys.env.contains("SPARK_HOME"), "spark.test.home or SPARK_HOME is not set.")
+      if (!(sys.props.contains("spark.test.home") ||  sys.env.contains("SPARK_HOME"))) {
+        fail("spark.test.home or SPARK_HOME is not set.")
+      }
       sys.props.getOrElse("spark.test.home", sys.env("SPARK_HOME"))
     }
```

Comment:
For reuse, could you move the logic to get `sparkHome` into `TestUtils`?

Reply:
Yeah, let's do that. There look two more occurrences.

```
sql/core/src/test/scala/org/apache/spark/sql/ExpressionsSchemaSuite.scala:      assert(sys.props.contains("spark.test.home") ||
sql/core/src/test/scala/org/apache/spark/sql/IntegratedUDFTestUtils.scala:    assert(sys.props.contains("spark.test.home") ||
```

Reply:
Ah, looks nice.

Reply:
Thanks @maropu @HyukjinKwon for the suggestion. I will make the change.